CHANGES.md

```diff
@@ -1,3 +1,129 @@
+### v0.15.1
+
+-  **TreeController**
+
+   Updates to the new version.
+
+-  **Component**
+
+   Makes creating Component’s templateObjects lazy and on-demand.
+   Reduces memory consumption.
+
+-  **Core**
+
+   Fixes super infinite loop in observed getter/setters.
+
+-  **DocumentResources**
+
+   Fix a bug affecting Firefox that was allowing components to be entering the document before its CSS stylesheets were loaded.
+
+-  **EventManager**
+
+   Makes preparedForActivationEvents public.
+   Fixes a case where the first touch in an app would be lost.
+   Adds support of PointerEvents.
+   Adds support of devices with multiples pointer types. (mouse/touch).
+   PrepareForActivationEvents is called when a pointing device (mouse/touch) is moved over a component’s element.
+   Blocks emulated mouse events.
+   Adds an option `blocksEmulatedEvents` to disable the filtering of emulated mouse events.
+   Fixes an issue that was blocking the logic for prepareForActionEvents on Firefox.
+   Reduces transient string concatenation in bubble/capture identifier based method name lookup/building.
+
+-  **Repetition**
+
+   Uses the PressComposer for managing the selection.
+   Adds support of PointerEvents.
+   Adds support of devices with multiples pointer types. (mouse/touch).
+   Increase performance.
+
+-  **Flow**
+
+   Fixes an issue that was preventing selection when clicking for the first time on an iteration.
+
+-  **PressComposer**
+
+   Adds support of PointerEvents.
+   Adds support of devices with multiples pointer types. (mouse/touch).
+   Makes sure the PressComposer operate just when the main button is pressed.
+   Cancel a press when a scroll or a translate occur.
+
+-  **RangeController**
+
+   Enhance RangeController filter/sort binding speed.
+
+-  **Composer**
+
+   Composers load lazily their logic by default.
+   Reduces unnecessary allocations of empty functions.
+   Adds the property isLoaded.
+
+-  **Target**
+
+   Adds load/unload functions that will load composers.
+
+-  **TranslateComposer**
+
+   Support Pointer Events
+   Support devices with multiple pointer types.
+   Better stealing claimed pointers logic.
+   Fixes issues between composers that was claiming the same pointer.
+   Does not listener on wheel events by default.
+   Fixes bug between inner scroller Components. (Was not able to work)
+   Add unload function. (release listeners)
+   Backwards compatibility for the Flow Translate Composer.
+   Adds supports for horizontal “wheel” movement.
+   Stops to raise translate events when a movement is beyond a extreme (max/min Translate X/Y).
+   Performance improvements. (less translateStart raised when handling wheel event, less listeners and diverse improvements)
```

Comment:
Performance improvements (less translateStart *events sent* when handling wheel event, less listeners, and diverse improvements).